### DISCUSSION TITLE
Raspberry PI 3 B+ and Problems with sound output

### DISCUSSION THREAD

**Discussion by gerdb** [cardonabits/haxo-hw#37] [General]
Had problems with my setup (RPI3 B+). I  only heard some noise. Found out that the HDMI connector touches the headphone jack and causes a short circuit.

Solution: A piece of tape on top of the HDMI connector

![RPI_HDMI_Headphone_Jack](https://github.com/cardonabits/haxo-hw/assets/883695/8076e027-1651-414f-8968-11d3bf9dc8ad)

**Comment by jcard0na** [comment]
Oh, wow! Thanks for reporting and providing the picture and a workaround!
I have reproduced this as well, and confirmed that when the HDMI connector touches the mini jack port, a high pitch screeching sound can be heard on the headphones.  This is only an issue on the Raspberry Pi 3.  The Pi Zero is shorter, and the Pi 4 has a smaller HDMI port (micro HDMI) that has a lower profile.

**Reply by jcard0na** [reply]
Ah, and your workaround worked flawlessly! 
![image](https://github.com/cardonabits/haxo-hw/assets/676181/6b80a346-b3d2-474f-8d1d-224f23151093)
